sgx_quote/src/verify.rs

```diff
@@ -43,7 +143,76 @@ pub(crate) fn verify_pck_chain_and_tcb(
     let (_, root_ca_cert) =
         parse_x509_certificate(&chain[2]).map_err(|e| Error::X509ParserError(e.into()))?;
 
-    debug!("Verifying certificates signature...");
+    verify_pck_certs(&pck_cert, &pck_ca_cert, &root_ca_cert)?;
+
+    if tee_type == &IntelTeeType::Sgx {
+        debug!("Verifying root ca crl...");
+
+        let root_ca_crl = get_root_ca_crl(pccs_url)?;
+        verify_root_ca_crl(&root_ca_cert, &root_ca_crl)?;
+    } else {
+        let (qe_identity_issuer_chain, raw_qe_identity) = get_qe_identity(pccs_url, tee_type)?;
+        let (qe_identity, crl_distribution_points) =
+            verify_qe_identity(&qe_identity_issuer_chain, &raw_qe_identity, &root_ca_cert)?;
+
+        verify_qe_report(qe_report, qe_identity)?;
+
+        let mut all_error = true;
+        for crl_url in &crl_distribution_points {
+            let ca_crl = get_root_ca_crl_from_uri(crl_url);
+            match ca_crl {
+                Ok(ca_crl) => {
+                    all_error = false;
+                    verify_root_ca_crl(&root_ca_cert, &ca_crl)?;
```

Comment:
Do we need to find just one root CA CRL?
```suggestion
                    verify_root_ca_crl(&root_ca_cert, &ca_crl)?;
                    break;
```